src/index.ts

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+import * as yargs from 'yargs';
+import { initProject as createProject, initProjectHere as createProjectHere } from './create-project';
+import { openDocs as docs } from './docs';
+
+// tslint:disable-next-line
+yargs
+  .usage('Usage: kak-cli <command>')
+  .version()
+  .command(
+    ['create-project <dir> [version]', 'cp <dir> [version]'],
```

Comment:
It does not work as I wanted it to work. Project name is always required. So you cannot do `kakunin-cli create-project --dir /some/dir` you have to `kakunin-cli create-project my-project --dir /some/dir`